src/bot/commands/help.ts

```diff
@@ -1,18 +1,40 @@
 import Discord, { Collection } from 'discord.js';
-
 import ICommand from './ICommand';
+import ConfigService from '../../core/services/config.service'
+import container from '../../inversity.config';
 
+const prefix = container.resolve<ConfigService>(ConfigService).get('BOT_TRIGGER');
 const command: ICommand = {
   name: 'help',
   aliases: ['commands'],
-  description: 'Lists available commands.',
+  example: `\`${prefix}help ping\``,
+  description: 'Lists available commands!',
   async execute(message: Discord.Message, args: string[], prefix: string, commands: Collection<string, ICommand>) {
     const data = [];
 
     if (!args.length) {
       // Get for all commands
-      data.push('Here\'s a list of all my commands:\n');
-      data.push(commands.map(command => command.name).join(', '));
+      data.push('here\'s a list of all my commands:\n');
+
+      const cmds = commands.map(command => command.name);
+      var cmd;
+      cmds.forEach(element => {
+        cmd = commands.get(element) || commands.find(c => c.aliases && c.aliases.includes(element));
+        var response = `\`${prefix}${cmd.name}\` `;
+        if (cmd.description)
+        {
```

Comment:
Generally in JS it's common to have the opening brace of a conditional, function, etc on the same line.
So like
```
if (cmd.description) {
    ...
}
```